peneira.kak

```diff
@@ -81,7 +81,7 @@ define-command -hidden peneira-finder -params 4 %{
                 set-register p %%sh{
                     # Execute command that generates candidates, and populate temp file
                     file=$(mktemp)
-                    eval "%s" > $file
+                    eval '(%s)' > $file
```

Comment:
... and double-quotes this subshell:
```suggestion
                    eval "(%s)" > $file
```
Then your `peneira-git-files` examples will work as expected *and* we wouldn't need to wrap everything inside an `eval`, what (as I argue) would make `peneira`'s behaviour diverge from that of Kakoune itself.

Reply:
Please, tell me if that is enough to fix problems you pointed out.

Reply:
Whenever you use one of the `$kak_quoted_...` variables you need to use `eval`. The [shell expansions documentation](https://github.com/mawww/kakoune/blob/master/doc/pages/expansions.asciidoc#shell-expansions) has the following example

    eval set -- "$kak_quoted_selections"
    while [ $# -gt 0 ]; do
        # ... do a thing with $1 ...
        shift
    done

and explains

> The eval command will take the expanded `$kak_quoted_selections` and unquote them, then execute the resulting set command, which sets the shell’s argument variables to the items from `$kak_selections`

so I think requiring `eval` is consistent with Kakoune.